packages/api/src/gobin/src/sql_obfuscator.go

```diff
@@ -0,0 +1,39 @@
+package main
```

Comment:
Would be nice if we write a simple test for this script

Reply:
It's tested via the integration in sqlObfuscator.test.ts 👍 If we continue down this path and expand it in the future I expect it'll get its own repo